src/game.cpp

```diff
@@ -5385,7 +5385,7 @@ void game::draw_minimap()
 
 float game::natural_light_level( const int zlev ) const
 {
-    if( zlev > OVERMAP_HEIGHT || zlev < 0 ) {
+    if( zlev > OVERMAP_HEIGHT || zlev < 0 ) { // ignore while underground or above limits
```

Comment:
Please put this on the previous or next line, don't add comments to the end of statements.